client/state/products-list/selectors.js

```diff
@@ -1,3 +1,7 @@
+import { property } from 'lodash';
+
+export const getProductsList = property( 'productsList.items' );
```

Comment:
I'm absolutely not against testing. Here's what I'm trying to say though…

```js
import { getProductsList } from '../selectors';

const mockState = { productsList: { items: [] } };

expect( getProductsList( mockState ) ).to.eql( [] );
```

That's the test here, and I've seen many tests very much like this in Calypso. I don't want to add them because they don't actually test anything. They are literally just duplicating code and doubling our maintenance burden. How did we know what kind of data to mock out? By looking at the implementation of the function we're trying to test. It's a circular dependency and it's a net loss for maintainability.

Put in other words, what happens if we were to change the structure of the state tree? Our test wouldn't fail. The test is built around the structure of the existing code. To make the change successful we need to start by updating both the test and the logic together.

Granted, this is not to say that we can't theoretically test things here but I would argue that we're not setup for it and to set ourselves up to properly test would require far more effort than this PR warrants.